Mage.Sets/src/mage/cards/a/AngelOfSerenity.java

```diff
@@ -55,7 +55,7 @@ public AngelOfSerenity(UUID ownerId, CardSetInfo setInfo) {
         this.addAbility(ability);
 
         // When Angel of Serenity leaves the battlefield, return the exiled cards to their owners' hands.
-        this.addAbility(new LeavesBattlefieldTriggeredAbility(new ReturnFromExileForSourceEffect(Zone.HAND, false, true), false));
+        this.addAbility(new LeavesBattlefieldTriggeredAbility(new ReturnFromExileForSourceEffect(Zone.HAND).withText(true, true), false));
```

Comment:
Interesting use case to split a card logic settings (constructor params) and a text generation settings (withText params).